braid/access.c

```diff
@@ -95,7 +95,7 @@ _braid_FAccess(braid_Core     core,
          if ( access_level >= 1 )
          {
             _braid_AccessStatusInit( ta[ci-ilower], ci, rnorm, iter, level, nrefine, gupper,
-                                     done, 0, braid_ASCaller_FAccess, astatus);
+                                     done, 0, braid_ASCaller_FAccess, u->basis, astatus);
```

Comment:
I'm guessing this is how you give the user access to the basis at this point?

Reply:
Yes, this prevents having to add another argument to the user's step function that they will potentially never use.